Pear/Cells/ChatTableViewCell.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ChatTableViewCell.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 5/9/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class ChatTableViewCell: UITableViewCell {
+
+    // MARK: - Private View Vars
+    private var chatBubble = UIView()
+    private var chatMessage = UILabel()
+    private var pairProfilePic = UIImageView()
+
+    // MARK: - Private Data Vars
+    static let reuseId = "chatReuseIdentifier"
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        contentView.backgroundColor = .backgroundLightGreen
+
+        chatBubble.layer.cornerRadius = 16
+        chatBubble.backgroundColor = .red
+        chatBubble.clipsToBounds = true
+        contentView.addSubview(chatBubble)
+
+        chatMessage.font = ._16CircularStdBook
+        chatMessage.numberOfLines = 0
+        chatMessage.textColor = .black
+        chatBubble.addSubview(chatMessage)
+
+        pairProfilePic.contentMode = .scaleAspectFill
+        pairProfilePic.layer.cornerRadius = 21
+        pairProfilePic.clipsToBounds = true
+        contentView.addSubview(pairProfilePic)
+
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupConstraints(message: PearMessage, currentUserNetID: String) {
+
+        let messagePadding: CGFloat = 5
+        let rightMessagePadding: CGFloat = -15
```

Comment:
we probably do not need padding variables for left and right if they are only going to be used once!